app/controllers/organisation/consultations_controller.rb

```diff
@@ -96,6 +117,23 @@ def publish
 		redirect_back fallback_location: root_path,  flash_success_info: "Consultation was successfully approved."
 	end
 
+  def extend_deadline
+    @consultation.update(secure_params)
```

Comment:
@BALAJI-BALA - make `extend_deadline` into it's own method in the model. We might have some actions like triggering emails.